google-cloud-spanner/benchmark/benchwrapper/benchwrapper.rb

```diff
@@ -29,28 +29,28 @@
 class ServerImpl < SpannerBench::SpannerBenchWrapper::Service
   def initialize()    
     @spanner = Google::Cloud::Spanner.new project: "someproject"
-    @client = spanner.client "someinstance", "somedatabase"
+    @client = @spanner.client "someinstance", "somedatabase"
   end
 
   def read(read_query, _call)
-    client.execute(read_query.Query).rows.each do |row|
+    @client.execute(read_query.query).rows.each do |row|
       # Just iterate over all rows.
     end
     EmptyResponse.new
   end
 
   def insert(insert_query, _call)
-    client.commit do |c|
-      c.insert "sometable", insert_query.users.map { |user| {name: user.name, age: user.age} }
+    @client.commit do |c|
+      c.insert "Singers", insert_query.singers.map { |singer| {SingerId: singer.id, FirstName: singer.first_name, LastName: singer.last_name} }
```

Comment:
Break up this line a bit for readability. (The hash inside the `map` block could be split onto multiple lines, and maybe create a temporary local variable for the result of the `map` call so the block isn't syntactically ambiguous regarding whether it belongs with `map` or `insert`.)